src/pages/journal/journal.scss

```diff
@@ -10,6 +10,22 @@ page-journal {
   .refresh-button-icon {
     padding: 0 10px;
   }
+  #unauthenticated-mode-indicator {
+    text-align: center;
+    padding: 16px 0;
+    background: map-get($colors-gds, 'gds-red');
+    #unauth-text {
+      color: #FFFFFF;
```

Comment:
Do we not already have a variable for the colour white?

Reply:
have replaced with the already existent white.